src/SourceLocator/Type/AnonymousClassObjectSourceLocator.php

```diff
@@ -47,6 +47,8 @@
      *
      * @throws \InvalidArgumentException
      * @throws \ReflectionException
+     *
+     * @psalm-suppress DocblockTypeContradiction
```

Comment:
Adding custom annotations is a bit risky due to various annotation parsers. @muglug what do you think about these? Is this kind of stuff something that `psalm.xml` can handle at all?